prime-router/metadata/hl7_mapping/STLTs/CO/CO.yml

```diff
@@ -0,0 +1,65 @@
+name: CO
+hl7Type: ORU_R01
+hl7Version: 2.5.1
+extends: ../../ORU_R01/ORU_R01-base
+elements:
+  # Source: ReportStream CO Requirements from settings
+  - name: receiving-application-namespace-id
```

Comment:
What's the purpose of having this name and value if it doesn't have an hl7Spec or isn't referenced elsewhere in the ORU_R01 schema?

Reply:
Wait I take it back! I researched and found it referenced in MSH my bad!

Reply:
This one is overwriting an existing element in the parent schema